packages/design-system/src/components/dropDown/select/components.js

```diff
@@ -91,27 +91,30 @@ SelectButton.propTypes = {
   isOpen: PropTypes.bool,
 };
 
-export const ChevronWrap = styled.div(
-  ({ theme, isOpen }) => css`
-    color: ${theme.colors.fg.secondary};
-    width: 32px;
+const DisclosureIcon = styled(ChevronDownSmall)(
+  ({ theme }) => css`
     height: 32px;
+    width: auto;
+    color: ${theme.colors.fg.secondary};
+    transition: transform 100ms;
 
-    ${isOpen &&
-    css`
+    &.open {
       transform: rotate(180deg);
-    `}
+    }
```

Comment:
We don't normally use class names with S-C.

We would do this using either a property:

```js
const DisclosureIcon = styled(ChevronDownSmall)(
  ({ theme }) => css`
  ...
  ${({ isOpen}) => isOpen && 'transform: rotate(.5turn);'}
`;
```

Or two different elements:

```js
const DisclosureIconOpen = styled(ChevronDownSmall)...;
const DisclosureIconClosed= styled(DisclosureIconOpen)`
  transform: rotate(.5turn);
`;
```

Or just an inline style directly on the element:

```js
<DisclosureIcon style={isOpen ? {transform: 'rotate(.5turn)'} : {}} />
```

Or the new fourth option we've been using in a few places: CSS Variable:


```js
const DisclosureIcon = styled(ChevronDownSmall)(
  ({ theme }) => css`
  ...
  --chevron-rotation: 0;
  transform: rotate(var(--chevron-rotation));'}
`;
...
<DisclosureIcon style={isOpen ? {'--chevron-rotation': '.5turn'} : {}} />
```
I think we should stick to one of those and not start mixing in a new style like this.  I think all of the above can be used for this particular instance. The first method would cause a new style to be injected into the DOM the first time a button is opened, but after that, it would reuse existing classes.

I can see how this class name based approach would make sense though, as it is pretty clean and fast, it is just not the way we do this normally across all the existing components.